### PR TITLE
Replicants documentation update

### PR DESCRIPTION
**Pull request by georgeoobrienNCAS**
Development branch set up.
Touched up some documentation.
Sending you this to remind you to have a look at it when you get the chance

### CODE REVIEW

File: cfstore/cfin.py [hunk 1]
```diff
@@ -154,28 +154,17 @@ def getBMetadata(ctx, arg1, argm):
...
     #Add collection, I guess?
-    x.add_collection(path, collection)
+#    x.add_collection(path, collection)
 
     #Set up something that runs on Jasmin
     # Connect to remote host
-    client = paramiko.SSHClient()
-    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-    client.connect(host, username=user)
+    x.getBMetadata(path, collection, False, False,None)
```
Comment (bnlawrence) [inline cfstore/cfin.py:161]: There doesn't seem to be any sort of documentation following the new method?

**Comment by bnlawrence** [comment]
I'm a bit puzzled insofar as the pull request seems to include partially functioning b-metadata stuff? Is that right?

**Comment by georgeoobrienNCAS** [comment]
> I'm a bit puzzled insofar as the pull request seems to include partially functioning b-metadata stuff? Is that right?
The commits before the request were intentional. The commits after (the partially functioning b-metadata stuff) were me pushing to a branch with an active pull branch (i.e. me doing something silly). 
I'll split the intended changes off into a branch. That makes more sense - single issue branches with a pull request is best practice for a reason, I suppose.

**Review by bnlawrence — APPROVED** [review]
As discussed, we'll get this in, and then be a bit more clinical from here on in.

**Pull request merged**